packages/core/src/color-mix.ts

```diff
@@ -2,6 +2,7 @@ import type { TransformArgs } from '@pandacss/types'
 
 export const colorMix = (value: string, token: TransformArgs['token']) => {
   if (!value) return { invalid: true, value }
```

Comment:
I believe we can remove this line now. Since the typeof check below will also match